tests/unit/src/test/scala/org/scalafmt/tests/CoverageSuite.scala

```diff
@@ -0,0 +1,253 @@
+// format: off
```

Comment:
(maybe ironcially) I am quite negative on escape hatch comments like this. The setting `align = more` does align comments as is done in this file. What do you think about enabling `align = more` so we can avoid `// format: off`? I just gave it a try and a large number of pattern matches become aligned and I think I can live with it.